CHANGELOG.md

```diff
@@ -10,6 +10,7 @@ and this project adheres to the versioning scheme outlined in the [README.md](RE
 ### Added
 
 - Add `vrf_seed` to the `/v3/sortitions` rpc endpoint
+- Add miner configuration option `tenure_extend_buffer_secs` to specify the number of seconds of buffer the miner should wait, after reaching the 70% threshold for tenure extension time, before issuing a tenure extension, to allow for clock skew between the miner and signers
```

Comment:
This is okay to me, but I was thinking we should enable this via the _signer set_ rather than the miners.

The signers themselves would have a `tenure_extend_buffer_secs` config variable, which they applied to the deadlines that they report (but not to the deadline that they check). This would allow the signer set to upgrade and configure this behavior independent of what any miners are doing.

Reply:
Ah, ok. Yeah, maybe that is better. I'll update this PR.

Reply:
Moved to signer in 156e11de1d7b96bba2da1416a3ed08e0cf6abe3e